pkg/application/vault/options.go

```diff
@@ -0,0 +1,46 @@
+package vault
+
+import (
+	"github.com/awslabs/eksdemo/pkg/application"
+	"github.com/awslabs/eksdemo/pkg/cmd"
+)
+
+type AppOptions struct {
+	application.ApplicationOptions
+	EnableTLS bool
+	Replicas  int
+}
+
+func newOptions() (options *AppOptions, flags cmd.Flags) {
+	options = &AppOptions{
+		ApplicationOptions: application.ApplicationOptions{
+			DefaultVersion: &application.LatestPrevious{
+				LatestChart:   "0.28.0",
+				Latest:        "v1.16.1",
+				PreviousChart: "0.27.0",
+				Previous:      "v1.15.2",
+			},
+			Namespace: "vault",
+		},
+		Replicas: 1,
+	}
+
+	flags = cmd.Flags{
+		&cmd.BoolFlag{
+			CommandFlag: cmd.CommandFlag{
+				Name:        "enable-tls",
+				Description: "Enable TLS for end-to-end encrypted transport",
+			},
+			Option: &options.EnableTLS,
+		},
+		&cmd.IntFlag{
+			CommandFlag: cmd.CommandFlag{
+				Name:        "replicas",
+				Description: "Number of replicas (3 recommended)",
```

Comment:
```suggestion
				Description: "Number of replicas (3 for HA)",
```
Since this is `eksdemo` we don't need to recommend 3 replicas. I think letting users know they should use 3 for HA is valuable since it's RAFT and it's not HA with 2.

Reply:
Modified slightly from your suggestion:
`Number of replicas (3 recommended for High Availability)`